src/web/app/components/sessions-table/sessions-table.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="row" *ngIf="sessionsTableRowModels.length > 0; else noSessionMessage">
```

Comment:
If we were to move the `ngIf` logic here to `<table>`, we might be able to remove both `<div>`s entirely. Let's try this and see how it goes.

Reply:
@agenell Do implement this change as well.